plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -294,6 +304,10 @@ public Instance startService(String namespace,
 
             docker.startContainer(StartContainerParams.create(container));
 
+            if (!Strings.isNullOrEmpty(cheUserId)) {
```

Comment:
Can you also cover in test that if property is not set then user is not set on container creation and exec is not called.